api/v1alpha1/network_types.go

```diff
@@ -108,6 +108,10 @@ type NetworkResourceSpec struct {
 	// +listType=set
 	// +optional
 	AvailabilityZoneHints []AvailabilityZoneHint `json:"availabilityZoneHints,omitempty"`
+
+	// projectRef is a reference to the ORC Project this resource is associated with.
```

Comment:
I might add something here about only being useful for admin, and that tenants should leave it empty to get their default.

Reply:
Good point. I've made it clear it's supposed to be used by admins only.